core/types/transaction.go

```diff
@@ -55,31 +56,24 @@ type Transaction struct {
 type inner interface {
 	// ChainId returns which chain id this transaction was signed for (if at all)
 	ChainId() *big.Int
-
 	// Protected returns whether the transaction is protected from replay protection.
 	Protected() bool
-
-	// MarshalJSONWithHash marshals as JSON with a hash.
+	// MarshalJSONWithHash marshals as JSON with a given hash.
 	MarshalJSONWithHash(hash *common.Hash) ([]byte, error)
-
 	// UnmarshalJSON unmarshals from JSON.
 	UnmarshalJSON(input []byte) error
-
 	// AccessList returns the transactions optional EIP-2930 access list.
 	AccessList() *AccessList
-
```

Comment:
hey @lightclient could you please re-lint this file? It will be accepted as is, but the default linter will add some whitespaces here which means the next one to modify this file will have a bunch of changes that make git blaming harder

Reply:
Hi @MariusVanDerWijden, I've rerun the linter (`make lint`), but am not seeing any changes? Is that the correct way to re-lint?

Reply:
I think what he meant is: please run `gofmt -w -s core/types/transaction.go`

Reply:
Hmm after running, I'm still not seeing any changes. Is there something specific you are concerned with that I should modify?